examples/python/differentiable_blocks_world/README.md

```diff
@@ -40,6 +40,4 @@ To stabilize the optimization and avoid local minima, a 3-stage optimization is
 
 https://vimeo.com/865329177?autoplay=1&loop=1&autopause=0&background=1&muted=1&ratio=10000:8845
 
-Check out the [project page](https://www.tmonnier.com/DBW/), which also contains examples of physical simulation and scene editing enabled by this kind of scene decomposition.
```

Comment:
The project page seems gone, so I removed the link to get the linkchecker green again